PeopleCounterApi/Message/Commands/PeopleCounter/PeopleCounterAddCommand.cs

```
@@ -21,10 +23,14 @@ public class PeopleCounterAddCommand : ICommand, ICommandFactory
 
         public async Task Execute()
         {
-            PeopleCounterViewModel peopleCounterView = JsonConvert.DeserializeObject<ViewModels.PeopleCounterViewModel>(_messageModel.Message);
-            PeopleCounterCore.Models.PeopleCounter peopleCounter = Mapping.MapPeopleCounter(peopleCounterView);
-
-            await _peopleCounterService.AddPeopleCounter(peopleCounter);
+            IEnumerable<PeopleCounterGetViewModel> peopleCounterView = JsonConvert.DeserializeObject< IEnumerable<PeopleCounterGetViewModel>>(_messageModel.Message);
+            PeopleCounterCollection peopleCounters = Mapping.MapPeopleCounter(peopleCounterView);
+            
+            await _peopleCounterService.AddPeopleCounter(peopleCounters);
+            if (!peopleCounters.Any())
+            {
+                throw new ApplicationException($"PeopleCounters could not be added, PeopleCounter received from Rabbit was: {JsonConvert.SerializeObject(_messageModel.Message)}");
```

Make sure to handle this exception higher up, don't let it propagate back to Rabbit